pgex/examples/simple_projectile/main.py

```diff
@@ -0,0 +1,49 @@
+"""
+This file is a part of the 'Pygame Examples (pgex)' source code.
+The source code is distributed under the MIT license.
+"""
+
+import asyncio
+
+import pygame
+
+from .bullet import Bullet
+
+WIDTH, HEIGHT = 640, 360
+FPS = 60
```

Comment:
These constants could be stored in a dedicated file (`constants.py`, `config.py` etc.).